index.html

```diff
@@ -14,6 +14,7 @@
     <div class="tl-btn-group">
       <a href title="Audio toggle" class="item audio-toggle"><i class="fa fa-2x fa-volume-up"></i></a>
       <a href title="Speech toggle" class="item speech-toggle"><i class="fa fa-2x fa-assistive-listening-systems"></i></a>
+      <a href title="Visual toggle" class="item visual-toggle"><i class="fa fa-2x fa-eye"></i></a>
```

Comment:
@willwade Do you know if these toggles absolutely need  to be wrapped in a `<a>` tag rather than, say, a `<span>` tag? If so, is that for accessibility? See my comment here: https://github.com/AceCentre/morse-learn/pull/11/files#diff-f5ada8f6fca9fbaff2fd978bbd3dcd76R98.

Reply:
Hmm. I can’t see why really. Yes - my only thought is accessibility but even that I think we can overcome some if that if that’s the case by giving it a title 

I say ditch the a tag :)